dojo/api_v2/views.py

```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
         # include current month
         months_between += 1
 
-        endpoint_monthly_counts = get_period_counts_legacy(findings.qs.order_by('numerical_severity'), findings.qs.order_by('numerical_severity'), None,
-                                                            months_between, start_date,
-                                                            relative_delta='months')
+        endpoint_monthly_counts = get_period_counts_legacy(
+            findings.qs.order_by("numerical_severity"),
+            findings.qs.order_by("numerical_severity"),
+            None,
+            months_between,
+            start_date,
+            relative_delta="months",
+        )
 
     elif type(obj).__name__ == "Product":
         product = obj
 
         report_name = "Product Report: " + str(product)
-        report_title = "Product Report"
-        report_subtitle = str(product)
-        findings = ReportFindingFilter(request.GET, product=product, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product=product)))
+        str(product)
```

Comment:
```suggestion

```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb